README.md

```diff
@@ -132,10 +132,15 @@ Currently the different available records are
 
 ``` js
 {
-  data: Buffer('some text')
+  data: Buffer(...)
+}
+{
+  data: 'some string'
```

Comment:
Should be `data: [Buffer('some text'), Buffer('more text')]`

Reply:
Also, add a note that the decoded value is always in the format `[Buffer('some text')]`

Reply:
Good catch! Forgot to update the doc.